site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
```

Comment:
I think we can remove this line, given that there is already a link above.

But we should add:

The intended audience for this page are [batch users](https://kueue.sigs.k8s.io/docs/tasks#batch-user).